app/representers/cart_item_representer.rb

```diff
@@ -0,0 +1,10 @@
+module CartItemRepresenter
+  include Roar::JSON::HAL
+
+  property :type, getter: ->(a) {'cart_item'}, writeable: false
```

Comment:
Unused block argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.